src/Audio_Registers.md

```diff
@@ -0,0 +1,375 @@
+# Audio Registers
+
+Audio registers are named following a `NRxy` scheme, where `x` is the channel number (or `5` for "global" registers), and `y` is the register's ID within the channel.
+Since many registers share common properties, a notation is often used where e.g. `NRx2` is used to designate `NR12`, `NR22`, `NR32`, and `NR42` at the same time, for simplicity.
+
+As a rule of thumb, for any `x` in `1`, `2`, `3`, `4`:
+- `NRx0` is some channel-specific feature (if present),
```

Comment:
Just state explicitly what it's used for, and that it's unused for the other two channels. Don't worry too much that this section must be completely generalized in its language at all cost.

Reply:
Considering that the list is all about "the rule of thumb", being generic is the name of the game. The goal of this paragraph is to give the reader a simple flowchart to quickly sort "uh, what *exactly* is NRxy?" (y = 1 ⇒ sound length, y = 2 ⇒ volume, y = 3 ⇒ freq low, y = 4 ⇒ freq hi + trigger)

Reply:
How would you word it?